File: Práctica_7/ejercicios_js/calculos.js
const MAX = 10000000;
let i = 0;
let seno = 0, coseno = 0, tangente = 0, logaritmo = 0, raiz = 0;
while (i++ < MAX) {
    seno += Math.sin(i);
    coseno += Math.cos(i);
    tangente += Math.tan(i);
    logaritmo += Math.log(i);
    raiz += Math.sqrt(i);
}

File: Práctica_7/ejercicios_c/calculos.c
#include <math.h>
int main(void){
	float lo=0 ,s=0, c=0, t=0, r2=0;
	double i;
	int max=10000000;
	while (i < max){

		s+=sin(i);
		c+=cos(i);
		t+=tan(i);
		lo+=log(i);
		r2+=sqrt(i);
		i++;
	}

	return (0);
}
